pennylane/fermi/fermionic.py

```diff
@@ -242,3 +242,88 @@ def simplify(self, tol=1e-8):
         for fw, coeff in items:
             if abs(coeff) <= tol:
                 del self[fw]
+
+
+# pylint: disable=too-few-public-methods
+class FermiC(FermiWord):
+    r"""FermiC(wire)
+    The fermionic creation operator :math:`a^\dagger`
+
+    **Details:**
+
+    * Number of wires: 1
+    * Number of parameters: 0
+
+    Args:
+        wire(int): the non-negative integer indicating the state the operator acts on.
+
+    For instance, ``qml.FermiC(3)`` (:math:`a^\dagger_3`) operating on :math:`\ket{0000}` gives
+    :math:`\ket{0001}`.
+
+    .. note:: While the ``FermiC`` class represents a mathematical operator, it is not a PennyLane ``Operator``
+
+    .. seealso:: :class:`~pennylane.FermiA`
+
+    **Example**
+
+    To construct the operator :math:`a^\dagger_0`:
+
+    >>> FermiC(0)
+    <FermiWord = '0+'>
+
+    This can be combined with the annihilation operator :class:`~pennylane.FermiA`. For example,
+    :math:`a^\dagger_0 a_1 a^\dagger_2 a_3` can be constructed as:
+
+    >>> qml.FermiC(0) * qml.FermiA(1) * qml.FermiC(2) * qml.FermiA(3)
+    <FermiWord = '0+ 1- 2+ 3-'>
+    """
+
+    def __init__(self, wire):
+        if not isinstance(wire, int) or wire < 0:
+            raise ValueError(
+                f"FermiC: expected a single, positive integer value for wire, but received {wire}"
+            )
```

Comment:
Do we need this wire validation anywhere in the fermi pipeline? I was expecting anything that is of the `Wires` type class to be valid. E.g., `qml.PauliX(-1)` is fine.

Reply:
Yes, I think we should add it to `FermiWord` as well. In fact, what we call wires are `orbital` indices not wires, so allowing arbitrary naming is a bit tricky. If we do allow, for instance -1, then we need extra information about orbital ordering. We thought about it and decided to just accept positive numbers as orbital/wire indices. However, open for suggestions.